src/adapter/src/coord/appends.rs

```diff
@@ -520,7 +520,13 @@ impl Coordinator {
             .metrics
             .append_table_duration_seconds
             .with_label_values(&[]);
-        info!("Appending to tables at {timestamp}, advancing to {advance_to}");
+        let modified_tables: Vec<_> = appends
+            .iter()
+            .filter_map(|(id, updates)| if !updates.is_empty() { Some(id) } else { None })
```

Comment:
We could further filter out system tables because there are a handful of those that are written on a periodic basis.

Reply:
If I just simply start up Materialize locally, I'm not seeing any system table inserts that would be happening all the time. But I'm not sure whether there would be some of those with more objects. Maybe we write stuff periodically when there are sources or something. So, I'd sleep better if we filtered out system tables.